wdl-lint/tests/lints/missing-comment-space/source.wdl

```diff
@@ -1,5 +1,7 @@
 ##This preamble comment is missing a space.
-## But this one isn't!
+##
+## But this one isn't and neither are the empty ones
+##
```

Comment:
O this is a potentially interesting case. Should we allow line 4? I'm not sure. In the [future] Markdown conversion, (depending on implementation specifics) this could cause an "extra" blank line to be printed between elements. But I don't think that really matters, does it? What about a more extreme case?
```
##
##
##
##
##

version 1.1
```
Should any of those lines get a lint?
Similar case I just thought of:
```
##
##
##
## Those are a bunch of meaningless blank(ish) lines.
## Are they a problem worth linting?

version 1.0
```

Reply:
I think both cases are harmless. They aren't ideal, but I don't think they're worth the hassle to lint. But my convictions here are weak.

Reply:
Personally, I think the lint is prescriptive enough; if the blank lines do end up being rendered differently in documentation, I would err on the side of leaving it be and hope the user has a reason for having them.

At the very least, the fact that the lines are all `## ...` make it look like a consistent and intentional preamble.

Reply:
Yea sounds good. No change needed!